lib/salus/scanners/osv/base.rb

```diff
@@ -0,0 +1,131 @@
+require 'salus/scanners/base'
+require 'json'
+require 'zip'
+require 'uri'
+require 'net/http'
+
+module Salus::Scanners::OSV
+  class Base < Salus::Scanners::Base
+    class ApiTooManyRequestsError < StandardError; end
+
+    MAX_RETRIES = 2
+
+    def run
+      raise NoMethodError
+    end
+
+    def should_run?
+      raise NoMethodError
+    end
+
+    private
+
+    def name
+      self.class.name.sub('Salus::Scanners::OSV::', '')
```

Comment:
What is this method supposed to do?
My guess is that if `self.class` is `A::B::C`, then `name` should return `"C"`?
If so, you might want to call `self.class.name.split('::')[-1]` instead, so that you don't need hardcode the class name (which could change).

Reply:
Yup, I don't think I use it so can remove it.